gh-pages/_docs/04-04-scmlogparser.md

```diff
@@ -53,22 +55,24 @@ You can also use the bash script anongit which generates an anonymous git log wi
 
 Standard usage:
 
-> `ccsh scmlogparser <log_file> --input-format [GIT_LOG|GIT_LOG_NUMSTAT|GIT_LOG_NUMSTAT_RAW|GIT_LOG_RAW|SVN_LOG]`
+> `ccsh svnlogparser <log_file> --input-format [SVN_LOG|GIT_LOG|GIT_LOG_NUMSTAT|GIT_LOG_NUMSTAT_RAW|GIT_LOG_RAW]`
 
 The result is written as JSON to standard out or into an output file (if specified by `-o` option).
 
 If a project is piped into the [SourceCodeParser]({{site.baseurl}}{% link _docs/04-02-sourcecodeparser.md %}), the results and the piped project are merged.
-The resulting project has the project name specified for the SCMLogParser.
+The resulting project has the project name specified for the SVNLogParser.
 
 ### Example using Git
 
+**Deprecated, use GitLogParser**
```

Comment:
Either rename the document to SvnLogParser and remove the git sections or create a new documentation for the new parser.